lib/galaxy/util/__init__.py

```diff
@@ -1194,6 +1194,25 @@ def unicodify(
     return value
 
 
+def decodify(value, encoding="utf-8", encoding_error="replace", decoding="latin-1", decoding_error="replace"):
```

Comment:
If we do decide for this I would narrow the type for value to string and really only do `value.encode(encoding).decode(encoding)`. The rest we can do when and if we need it. I would not like to replicate unicodify logic here.